examples/checklist.html

```diff
@@ -0,0 +1,156 @@
+<!doctype html>
```

Comment:
Thanks for this example - if you have several scenarios (single, second, third) please either combine all to a single JSON or split it to several example files.
In addition - make sure that all the examples work.
The goal of these examples is to allow developers to consider as many options as they can during development.